.github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+name: Tag and Release
+
+on:
+  push:
+    branches: ["main"]
+
+jobs:
+  release-on-push:
+    runs-on: ubuntu-22.04
```

Comment:
@okdas For consistency, is this what you're using now too?

Reply:
AFAICT poktroll is using `ubuntu-latest`. I agree with @commoddity tho that it is a better approach to fix the runner version. In this case, I'm using the same fixed version used in middleware.

See for reference:
https://github.com/pokt-network/poktroll/blob/3384138248474a2e724445384fbaff07ab0c41b2/.github/workflows/main-build.yml#L14
https://github.com/pokt-network/poktroll/blob/3384138248474a2e724445384fbaff07ab0c41b2/.github/workflows/release-artifacts.yml#L14